src/CalendarApp/CalendarApp.js

```diff
@@ -31,6 +31,48 @@ export default function CalendarApp() {
           />
         </main>
       </div>
+
+      <div className="calendarWorkoutContainer">
+        <div className="calendarWorkoutContainerContent">
+          <h3 className="calendarWorkoutTitle">Full Body Workout</h3>
+          <button className="calendarWorkoutButton">Start</button>
+
+          <div className="calendarExerciseList">
+            <div className="calendarExerciseItem">
+              <p className="itemDetailsName">Jumping Jacks</p>
+              <p className="itemDetailsReps">30 reps</p>
+            </div>
+            <div className="calendarExerciseItem">
+              <p className="itemDetailsName">Push-Ups</p>
+              <p className="itemDetailsReps">20 reps</p>
+            </div>
+            <div className="calendarExerciseItem">
+              <p className="itemDetailsName">Crunches</p>
+              <p className="itemDetailsReps">30 reps</p>
```

Comment:
I would componetise and use props but it can be a controvesial topic.

Reply:
I am creating a simple component with the mock data for now as per Ian's request. similar to what we did on Kodflix